simpleui-server/src/simpleui-server.ts

```diff
@@ -272,9 +334,14 @@ export class SimpleUIServer {
             app.get(`${propsFileQuery}`, async (req, res) => {
                 Logger.log(LogLevel.VERBOSE, `props request callback: ${++SimpleUIServer.requestCallbacks}`);
                 try {
-                    const props = PropsFileReader.getProps(
+                    let props = PropsFileReader.getProps(
                         `${req.params.propsStub}.properties`,
                         `${req.params.appName}`, cmdVars.webPort);
+
+                    if (cmdVars.override) {
```

Comment:
It looks like that 'override' thing is just a flag for if there are any overrides? If that's the case, you don't need it at all. You would just always make the call to 'addOverrides' with a possibly-empty object.

Reply:
fixed

Reply:
make sure to remove the 'override' flags hanging around

Reply:
oops forgot about them, removed them in my last commit